config/routes.rb

```diff
@@ -419,12 +419,12 @@
   get 'status', :to => 'home#status', :as => "status"
 
   # get only for alterator unattended scripts
-  get 'unattended/provision/:metadata', :controller => 'unattended', :action => 'host_template', :format => 'html',
+  get 'unattended/provision/:metadata', :controller => 'unattended', :action => 'host_template', :format => 'text',
     :constraints => { :metadata => /(autoinstall\.scm|vm-profile\.scm|pkg-groups\.tar)/ }
   # get for end of build action
-  get 'unattended/built/(:id(:format))', :controller => 'unattended', :action => 'built'
+  get 'unattended/built/(:id(:format))', :controller => 'unattended', :action => 'built', :format => 'text'
   # get for all unattended scripts
-  get 'unattended/(:kind/(:id(:format)))', :controller => 'unattended', :action => 'host_template'
+  get 'unattended/(:kind/(:id(:format)))', :controller => 'unattended', :action => 'host_template', :format => 'text'
```

Comment:
I think `set_content_type` in the UnattendedController is now obsolete by setting format here, it should be removed as it appears to be overriding this setting.